src/components/tables/EntityTable.tsx

```diff
@@ -455,118 +345,111 @@ function EntityTable({ noExistingDataContentIds }: Props) {
                                                             addSuffix: true,
                                                         }
                                                     )}
-                                                </TableCell>
+                                                </Box>
+                                            </Tooltip>
+                                        </TableCell>
 
-                                                <TableCell>
-                                                    <Box
-                                                        sx={{
-                                                            display: 'flex',
-                                                        }}
-                                                    >
-                                                        <Button
-                                                            variant="contained"
-                                                            size="small"
-                                                            disableElevation
-                                                            disabled
-                                                            sx={{ mr: 1 }}
-                                                        >
-                                                            Edit
-                                                        </Button>
-
-                                                        <Button
-                                                            variant="contained"
-                                                            size="small"
-                                                            color={
-                                                                deploymentStatus ===
-                                                                'ACTIVE'
-                                                                    ? 'error'
-                                                                    : 'success'
-                                                            }
-                                                            disableElevation
-                                                            disabled
-                                                        >
-                                                            {deploymentStatus ===
-                                                            'ACTIVE'
-                                                                ? 'Stop'
-                                                                : 'Run'}
-                                                        </Button>
-                                                    </Box>
-                                                </TableCell>
-                                            </TableRow>
-                                        )
-                                    )
```

Comment:
All the button code needs to be added back eventually but we need to figure out how we want to store/figure out status.